### PR TITLE
Coq Experimental Updates for Camera-Ready

### PR DESCRIPTION
**Pull request by alpaylan**
This PR allows for the Coq generators to be evaluated on the same set of tasks as the Haskell ones for BST and RBT workloads.

### CODE REVIEW

File: experiments/coq-experiments/5.1/Collect.py [hunk 1]
```diff
@@ -25,16 +23,15 @@ def collect(results: str):
             run_trial = None
 
             for strategy in tool.all_strategies(workload):
-                if strategy.name not in [
-                        'TypeBasedGenerator', 'BespokeGenerator', 'TypeBasedFuzzer',
-                        'SpecificationBasedGenerator'
-                ]:
+                if strategy.name in ['RookieGenerator']:
```
Comment (jwshii) [inline experiments/coq-experiments/5.1/Collect.py]: Can you undo this change please?

Comment (jwshii) [reply experiments/coq-experiments/5.1/Collect.py]: Or what is the RookieGenerator in this context?

File: workloads/Coq/RBT/Src/Impl.v [hunk 2]
```diff
@@ -24,8 +24,9 @@ Inductive Tree :=
 
 Derive (Show) for Tree.
 
-Axiom fuel : nat. Extract Constant fuel => "100000".
+(* Axiom fuel : nat. Extract Constant fuel => "100000". *)
```
Comment (jwshii) [inline workloads/Coq/RBT/Src/Impl.v]: Can you please remove this if it's outdated?

File: workloads/Coq/RBT/Strategies/BespokeGenerator.v [hunk 3]
```diff
@@ -15,47 +15,56 @@ From RBT Require Import Impl Spec.
...
+    end
+    in blacken_ (ins key val t).
+
+Definition gen_tree (s : nat) : G Tree :=
+    sz <- choose(1, s)%nat;;
+    kvs <- gen_kvs sz;;
+    ret (fold_right (fun '(k, v) t => insert_ k v t) E kvs).
+
+Definition bespoke :=
+    (* x <- choose (20, 100)%nat ;; *)
```
Comment (jwshii) [inline workloads/Coq/RBT/Strategies/BespokeGenerator.v]: Can you please remove this line?

File: workloads/Coq/RBT/Strategies/BespokeGenerator.v [hunk 4]
```diff
@@ -67,89 +76,90 @@ Global Instance genTreeSized : GenSized (option Tree) :=
 Definition gSized := 
     x <- choose (0, 3)%nat;;
     genTree 0 (2^(Z.of_nat(x)*2)) R fuel x
-.
+. *)
```
Comment (jwshii) [inline workloads/Coq/RBT/Strategies/BespokeGenerator.v]: Ditto

File: experiments/coq-experiments/5.1/Collect.py (hunk 1 above)
Comment (alpaylan) [reply experiments/coq-experiments/5.1/Collect.py]: It's the BST style generator for RBT, we don't use it in any of our trials, it was there as an early benchmark.

File: workloads/Coq/RBT/Strategies/BespokeGenerator.v (hunk 3 above)
Comment (alpaylan) [reply workloads/Coq/RBT/Strategies/BespokeGenerator.v]: Deleted

File: workloads/Coq/RBT/Src/Impl.v (hunk 2 above)
Comment (alpaylan) [reply workloads/Coq/RBT/Src/Impl.v]: Fixed.

File: experiments/coq-experiments/5.1/Collect.py (hunk 1 above)
Comment (alpaylan) [reply experiments/coq-experiments/5.1/Collect.py]: I can remove it altogether if you want. We don't use it anyway.

Comment (jwshii) [reply experiments/coq-experiments/5.1/Collect.py]: Oh ok, I'd rather enumerate the strategies that we do use than the ones that we don't because I find it easier to read. (I read this backwards at first). I will push a fix.

File: workloads/Coq/RBT/Strategies/BespokeGenerator.v (hunk 4 above)
Comment (alpaylan) [reply workloads/Coq/RBT/Strategies/BespokeGenerator.v]: Deleted

**Review by jwshii — APPROVED** [review]
